pkg/store/change/network/store.go

```diff
@@ -17,28 +17,26 @@ package network
 import (
 	"context"
 	"errors"
+	"io"
+	"time"
+
 	"github.com/atomix/go-client/pkg/client/indexedmap"
 	"github.com/atomix/go-client/pkg/client/primitive"
 	"github.com/atomix/go-client/pkg/client/util/net"
 	"github.com/gogo/protobuf/proto"
 	"github.com/google/uuid"
 	networkchange "github.com/onosproject/onos-config/api/types/change/network"
 	"github.com/onosproject/onos-config/pkg/config"
-	"github.com/onosproject/onos-config/pkg/store/cluster"
 	"github.com/onosproject/onos-config/pkg/store/stream"
 	"github.com/onosproject/onos-lib-go/pkg/atomix"
-	"io"
-	"time"
+	"github.com/onosproject/onos-lib-go/pkg/cluster"
 )
 
 const changesName = "network-changes"
 
-func init() {
-	uuid.SetNodeID([]byte(cluster.GetNodeID()))
-}
-
 // NewAtomixStore returns a new persistent Store
-func NewAtomixStore(config config.Config) (Store, error) {
+func NewAtomixStore(cluster cluster.Cluster, config config.Config) (Store, error) {
+	uuid.SetNodeID([]byte(cluster.Node().ID))
```

Comment:
I think it is safe to initialize it here instead initializing it at package level.